File: RK2/django_rest/urls.py
from django.urls import path
from . import views

urlpatterns = [
    path('', views.index),
    path('products/', views.products),
    path('companys/', views.companys),
    path('product/<int:id>/', views.product),
    path('company/<int:id>/', views.company),
]

File: lab7/src/modules/urls.js
class Urls {
    constructor() {
        this.url = 'http://127.0.0.1:8000/';
    }

    companys() {
        return `${this.url}Company/`;
    }

    companyId(id) {
        return `${this.url}Company/${id}`;
    }

    products() {
        return `${this.url}Product/`;
    }

    productId(id) {
        return `${this.url}Product/${id}`;
    }
}

export const urls = new Urls();
